api/.env.example

```
@@ -62,4 +62,10 @@ PORT=3000
 # Enables admin endpoints under /api/admin/crawler/*
 CRAWLER_ENABLED=false
 # Enables scheduled jobs (daily crawl + stale-source checks)
-CRAWLER_SCHEDULER_ENABLED=false
\ No newline at end of file
+CRAWLER_SCHEDULER_ENABLED=false
+# Maximum pages to crawl per session (default: 500)
+# CRAWLER_MAX_PAGES=500
+# Maximum crawl duration in milliseconds (default: 900000 = 15 minutes)
+# CRAWLER_MAX_DURATION_MS=900000
+# Delay between page fetches in milliseconds (default: 200ms, be respectful to servers)
+# CRAWLER_PAGE_DELAY_MS=200
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)